scripts/pi-hole/js/index.js

```diff
@@ -117,6 +117,86 @@ var customTooltips = function(tooltip) {
         tooltipEl.style.padding = tooltip.yPadding + "px " + tooltip.xPadding + "px";
 };
 
+var customPieTooltips = function(tooltip) {
```

Comment:
There are only minimal differences between this function and `customTooltips`. Merge them together so code is not duplicated.